lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -61,11 +61,18 @@ sub register {
         });
 
     $app->helper(
+        bug_report_actions => sub {
+            my ($c) = @_;
+            return $c->include_branding('external_reporting');
```

Comment:
no, it doesn't. For plain branding there is just a debug log output that there is no external reporting file in your branding or in plain branding